sal/class.json-api-site-base.php

```diff
@@ -507,4 +542,16 @@ function get_wordpress_version() {
 		global $wp_version;
 		return $wp_version;
 	}
+
+	protected function is_a8c_publication( $post_id ) {
```

Comment:
this should be split into a WPCOM method and Jetpack method, I assume `is_post_freshly_pressed` and `is_dailypost_blog` are WPCOM-only methods and you can remove those checks.

in `WPCOM_JSON_API_Site_Misc` (`trait.json-api-site-wpcom.php`):

``` php
protected function is_a8c_publication( $post_id ) {
  return is_post_freshly_pressed( $post_id ) || is_dailypost_blog( $this->blog_id );
}
```

In `class Jetpack_Site`:

```php
protected function is_a8c_publication( $post_id ) {
  return false;
}
```